woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2146,6 +2150,17 @@ def test_setitem_indexed_column_on_unindexed_dataframe(sample_df):
 
     assert sample_df.ww.index is None
     assert ww.is_schema_valid(sample_df, sample_df.ww.schema)
+    expected_tags = {}
+    if hasattr(sample_df.ww.logical_types['id'], 'standard_tags'):
+        expected_tags = sample_df.ww.logical_types['id'].standard_tags
+    assert sample_df.ww['id'].ww.semantic_tags == expected_tags
+
+    sample_df.ww.set_index('id')
+    col = sample_df.ww.pop('email')
+    col.ww.init(semantic_tags='index')
+    sample_df.ww['email'] = col
+    assert sample_df.ww.index == 'id'
+    assert 'index' not in sample_df.ww['email'].ww.semantic_tags
```

Comment:
In the same vein as the other comment about writing out the set of expected tags, we can kill a few birds with one stone by checking that `sample_df.ww['email'].ww.semantic_tags` matches a set of semantic tags explicitly written out. 

Saying `sample_df.ww['email'].ww.semantic_tags == set()` does the same thing but also would catch any unexpected tags that made their way in somehow

Reply:
Also, if you set email's logical type at init with `sample_df.ww.init(logical_types={'email':'Categorical')` then email will also have standard semantic tags that can be tested here